go.mod

```diff
@@ -4,5 +4,6 @@ go 1.16
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
`testify` probably not needed here - makes tests inconsistent (and makes it too easy to write tests without descriptive errors on failure).